broker-core/src/main/java/org/apache/qpid/server/transport/NonBlockingConnection.java

```diff
@@ -75,6 +76,7 @@
     private final AtomicLong _maxReadIdleMillis = new AtomicLong();
     private final List<SchedulingDelayNotificationListener> _schedulingDelayNotificationListeners = new CopyOnWriteArrayList<>();
     private final AtomicBoolean _hasShutdown = new AtomicBoolean();
+    private final AtomicInteger _loopingCounter = new AtomicInteger(0);
```

Comment:
The counter is only incremented in TLS delegate. Please move it into `NonBlockingConnectionTLSDelegate`